perllib/FixMyStreet/Script/ArchiveOldEnquiries.pm

```diff
@@ -73,6 +73,9 @@ sub close_list {
     die "Found more reports than expected\n" if $no_message->count + $with_message->count > $max_reports;
 
     $opts->{retain_alerts} = 1;
+
+    die "Aborting: the show_emails flag was specified. Run without --show_emails to close reports.\n" if $opts->{show_emails};
```

Comment:
I don't think this is needed because `close_problems` wouldn't do anything anyway? Preferable would be to die when options are being set up if both show_emails and reports are given, because providing both options together doesn't make sense. (And also die if both show_emails and commit are given, presumably.)